src/plugins/presentation_util/public/components/field_picker/field_picker.tsx

```diff
@@ -89,7 +89,8 @@ export const FieldPicker = ({
         ? uniq(
             dataView.fields
               .filter((f) => (filterPredicate ? filterPredicate(f) : true))
-              .map((f) => f.type as string)
+              .map((f) => getFieldIconType(f))
+              .filter((type) => isKnownFieldType(type))
```

Comment:
Not sure how it's supposed to work on Dashboards but I think it's better to keep all fields in the list even if we don't have an icon for them yet.

Reply:
100% - was going to say the same thing :) We should just handle these fields elegantly, like so:

![image](https://github.com/user-attachments/assets/7fd4d2d5-c536-4994-ad11-5a0b810ebb9d)

Reply:
@jughosta @Heenawter  fixed, filter removed

Em, There is a question that I am confused about, In `kbn-unified-field-list` -> `field_type_filter.tsx`, `availableFieldTypes` is filtered by `KNOWN_FIELD_TYPE_LIST`. 

And then unknown field type is also not shown in the field type filter list. Not sure if this is an optimization. 😃 

<img width="643" alt="image" src="https://github.com/user-attachments/assets/7253c6c2-6ccb-4cba-bcd1-7b6864ee9984">

Reply:
Yes, good point that Discover offers less field type options at the moment. I would not say though that it's a reason to remove it from Dashboard page, some users might already rely on it in their work flow.

Thanks for the applied changes!